change/@fluentui-react-datepicker-compat-39a831ff-e1f1-48df-af17-ffcd7fdf7883.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 5 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28188&commitId=b1796c351c87bb727e6624a44b87252f5ed3778b&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1939 screenshots unchanged, 0 screenshots with different dimensions and 5 screenshots with visible difference.


  <details><summary><b>unknown</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.Required.default.chromium.png|182| Changed|
|DatePicker Compat.Underlined and required.default.chromium.png|149| Changed|
|DatePicker Compat.Underlined.default.chromium.png|149| Changed|
|DatePicker Compat.With label.default.chromium.png|182| Changed|
|DatePicker Compat.allowTextInput.default.chromium.png|182| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
These are expected since they no longer have the focused styles.